velox/functions/lib/MapFromEntries.cpp

```diff
@@ -30,7 +30,10 @@ static const char* kIndeterminateKeyErrorMessage =
     "map key cannot be indeterminate";
 static const char* kErrorMessageEntryNotNull = "map entry cannot be null";
 
-// See documentation at https://prestodb.io/docs/current/functions/map.html
+// allowNullEle If true, will return null if map has
```

Comment:
Maybe use @tparam for template parameter.

> will return null if map has an entry with null as key or map is null

This comment looks not aligned with the documentation, could you double-check?